src/opm/parser/eclipse/EclipseState/SummaryConfig/SummaryConfig.cpp

```diff
@@ -116,19 +115,38 @@ namespace {
          {"SGAS" , {"BSGAS"}}
     };
 
+    using keyword_set = std::unordered_set<std::string>;
+
+    inline bool is_in_set(const keyword_set& set, const std::string& keyword) {
+        return set.find(keyword) != set.end();
+    }
+
     bool is_special(const std::string& keyword) {
-        using set = std::unordered_set<std::string>;
-        static const auto specialkw = set {
-            "NEWTON", "NAIMFRAC", "NLINEARS", "NLINSMIN", "NLINSMAX",
-            "ELAPSED", "MAXDPR", "MAXDSO", "MAXDSG", "MAXDSW", "STEPTYPE",
+        static const keyword_set specialkw {
+            "ELAPSED",
+            "MAXDPR",
+            "MAXDSG",
+            "MAXDSO",
+            "MAXDSW",
+            "NAIMFRAC",
+            "NEWTON",
+            "NLINEARS",
+            "NLINSMAX",
+            "NLINSMIN",
+            "RUNSUM",
```

Comment:
Observe that the `RUNSUM` keyword should not be in this list - it is special of a different kind:

1. The main rule is that the keywords are categorized after the first letter - e.g. variable starting with 'N' are generally *NETWORK* variables, whereas the keyword NEWTON is *not* a network variable - that is why *NETWORK* is in the special set. All the variables in this list should be output in the summary file - just in a slightly different way than their first letter should indicate.

2. RUNSUM is special in a different way. It does not represent a variable which should go to the summary at all, but rather a request to write the results in a `RSM` file.